src/bmaptool/CLI.py

```diff
@@ -478,7 +478,7 @@ def open_files(args):
             )
 
     # Try to open the destination file. If it does not exist, a new regular
-    # file will be created. If it exists and it is a regular file - it'll be
```

Comment:
Actually, in this case, the dash should be replaced with a 2nd comma. If the independent clause is introduced with a comma, it should be terminated with one too.